### ISSUE TITLE
Call to a member function connection() on null

### ISSUE THREAD

**Issue by Whyounes** [wouterj/WouterJEloquentBundle#30]
I keep getting this error when trying to run the migrations! This is the stack trace
```
  [Symfony\Component\Debug\Exception\FatalErrorException]
  Error: Call to a member function connection() on null


Exception trace:
 () at /vagrant/movies/vendor/illuminate/support/Facades/Schema.php:28
 Illuminate\Support\Facades\Schema::getFacadeAccessor() at /vagrant/movies/vendor/illuminate/support/Facades/Facade.php:144
 Illuminate\Support\Facades\Facade::getFacadeRoot() at /vagrant/movies/vendor/illuminate/support/Facades/Facade.php:231
 Illuminate\Support\Facades\Facade::__callStatic() at /vagrant/movies/app/migrations/2016_10_23_113543_create_users_table.php:19
 Illuminate\Support\Facades\Schema::create() at /vagrant/movies/app/migrations/2016_10_23_113543_create_users_table.php:19
 CreateUsersTable->up() at /vagrant/movies/vendor/illuminate/database/Migrations/Migrator.php:162
 Illuminate\Database\Migrations\Migrator->runUp() at /vagrant/movies/vendor/illuminate/database/Migrations/Migrator.php:130
 Illuminate\Database\Migrations\Migrator->runMigrationList() at /vagrant/movies/vendor/illuminate/database/Migrations/Migrator.php:97
 Illuminate\Database\Migrations\Migrator->run() at /vagrant/movies/vendor/wouterj/eloquent-bundle/src/Command/MigrateCommand.php:46
 WouterJ\EloquentBundle\Command\MigrateCommand->execute() at /vagrant/movies/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:256
 Symfony\Component\Console\Command\Command->run() at /vagrant/movies/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:837
 Symfony\Component\Console\Application->doRunCommand() at /vagrant/movies/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:187
 Symfony\Component\Console\Application->doRun() at /vagrant/movies/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:80
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /vagrant/movies/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:118
 Symfony\Component\Console\Application->run() at /vagrant/movies/bin/console:27
 {main}() at /vagrant/movies/bin/console:0
``

**Comment by wouterj** [comment]
Ah, I think you caught a major mistake in the `MigrationsMakeCommand` class. Thanks for reporting the error!

Instead of using `Illuminate\Support\Facades\Schema`, you should be using `WouterJ\EloquentBundle\Facade\Schema`. In the `MigrationCreater` class, I simply reuse the templates of laravel, but this of course includes the wrong use statements.

Expect a PR fixing this soon and thanks again for the report!

**Comment by wouterj** [comment]
While debugging this, I discovered the `Schema` use statement was introduced in Laravel 5.3: https://github.com/illuminate/database/commit/ef41824f4f7b6479f68084431825de5ecb87d489 dev-master only support Laravel 5.2 at the moment. So for now, I'm not going to fix this error.

However, as we'll support Laravel 5.3 after the next release, I'll keep this issue open to not forget to fix it.

**Comment by wouterj** [comment]
I'm verys sorry, seems I completely mess up everything. I already released 0.3.0, so it's time to fix Laravel 5.3 support.

Can you please confirm which version of this bundle you're running in your app at the moment? Is it 0.2.x or 0.3.x?

**Comment by Whyounes** [comment]
I'm currently using 0.2 I'll try to upgrade to 0.3 soon. Thanks for your help
